pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -106,6 +110,11 @@ func (c *Cluster) Start() error {
 		return fmt.Errorf("%v", err)
 	}
 
+	// fail if we were instructed to deploy more than one mgr on the same machine with host networking
+	if c.HostNetwork && c.allowMultiplePerNode && c.Replicas > 1 {
+		return fmt.Errorf("refusing to deploy %d managers on the same host since hostNetwork is %v and allowMultiplePerNode is %v. Only one manager per node is allowed", c.Replicas, c.HostNetwork, c.allowMultiplePerNode)
```

Comment:
We should check if the number of nodes available for mgr (that match the placement policy) as well. If there are actually multiple nodes available, it will be fine.

Reply:
Yes, you are right. When we set the count of mgr for host networking,  we must check the number of nodes available for mgr.